samples_project/Assets/SampleViewer/Samples/Measure/readme.md

```diff
@@ -2,13 +2,19 @@
 
 Draw lines on the map and show distance.
```

Comment:
`Measure the distance by drawing lines on the map.` or Display the measured distance by drawing lines on the map.`

"Draw lines" is not what this sample shows but "Measure" or " Display the measurement" is, right?